rmate

```diff
@@ -381,11 +390,32 @@ function handle_connection {
 
 # connect to textmate and send command
 #
-exec 3<> "/dev/tcp/$host/$port"
+if [ -n "$unix" ] ; then
+    if [ ! -S "$unix" ] ; then
+        echo "$unix is not a socket!"
+        exit 1
+    fi
+
+    fifodir=$(mktemp -d) && \
+        mkfifo "$fifodir/in" "$fifodir/out" && \
+        ( (nc -U "$unix" < "$fifodir/in" > "$fifodir/out" &) &) && \
```

Comment:
Not sure if I should be doing something smarter than nothing with the stderr of `nc`; maybe direct it to a file in the `fifodir` and cat it out if there's a connection error?